src/molecules/Hero/hero.module.css

```diff
@@ -0,0 +1,16 @@
+.hero {
+  padding: 20px 28px;
+  background-color: var(--hero-background-color);
+  color: var(--hero-text-color);
+}
+
+.hero h1 {
```

Comment:
Please arrange properties in alphabetical order.

Reply:
Done.